libmscore/scorefile.cpp

```diff
@@ -159,6 +159,8 @@ void Score::writeMovement(XmlWriter& xml, bool selectionOnly)
       xml.tag("showUnprintable", _showUnprintable);
       xml.tag("showFrames",      _showFrames);
       xml.tag("showMargins",     _showPageborders);
+      if (!_markIrregularMeasures)
+            xml.tag("markIrregularMeasures", _markIrregularMeasures);
```

Comment:
It only gets written to the score when not set to the default (i.e. when set to `false`, the default being `true`).
I wonder whether the other 3 settings above should also only get written in that case?

Reply:
Couldn't you just write
```
xml.tag("markIrregularMeasures", _markIrregularMeasures, true);
```
and achieve the same result? You can leave off the third parameter when the default value is `false` (or any other interpretation of `QVariant()`).

Reply:
Good idea. Doesn't save much though, only now the comparison is done somewhere else (in `xml.tag()`). And still no good way to determine the default, except to look into the constructor.

Reply:
Well, the comparison is done in `xml.tag()` anyway, which I guess was my point.